server/routes/contacts/manage/prisoner-search/prisonerSearchController.ts

```diff
@@ -0,0 +1,26 @@
+import { Request, Response } from 'express'
+import { PageHandler } from '../../../../interfaces/pageHandler'
+import { Page } from '../../../../services/auditService'
+import { PrisonerSearchSchemaType } from './prisonerSearchSchema'
+
+export default class PrisonerSearchController implements PageHandler {
+  public PAGE_NAME = Page.PRISONER_SEARCH_PAGE
+
+  GET = async (req: Request, res: Response): Promise<void> => {
+    const { journeyId } = req.params
+    const journey = req.session.manageContactsJourneys[journeyId]
+    const search = res.locals?.formResponses?.search ? res.locals?.formResponses?.search : journey?.search?.searchTerm
```

Comment:
This can be shorten by using 'optional chaining with (??)' `const search = res.locals?.formResponses?.search ?? journey?.search?.searchTerm;`

Reply:
Yes, good idea. Now done in next PR.